bindings/python/iota_sdk/types/address.py

```diff
@@ -19,67 +22,70 @@ class AddressType(IntEnum):
     NFT = 16
 
 
+@json
 @dataclass
 class Address():
     """Base class for addresses.
     """
     type: int
 
-    def as_dict(self):
-        return {k: v for k, v in self.__dict__.items() if v is not None}
-
 
+@json
 @dataclass
 class Ed25519Address(Address):
     """Represents an Ed25519 address.
     Attributes:
-        pubKeyHash: The hex encoded Ed25519 public key hash.
+        pub_key_hash: The hex encoded Ed25519 public key hash.
     """
-    pubKeyHash: HexStr
+    pub_key_hash: HexStr
     type: int = field(
         default_factory=lambda: int(
             AddressType.ED25519),
         init=False)
 
 
+@json
 @dataclass
 class AliasAddress(Address):
     """Represents an Alias address.
     Attributes:
-        aliasId: The hex encoded alias id.
+        alias_id: The hex encoded alias id.
     """
-    aliasId: HexStr
+    alias_id: HexStr
     type: int = field(
         default_factory=lambda: int(
             AddressType.ALIAS),
         init=False)
 
 
+@json
 @dataclass
 class NFTAddress(Address):
     """Represents an NFT address.
     Attributes:
-        nftId: The hex encoded NFT id.
+        nft_id: The hex encoded NFT id.
     """
-    nftId: HexStr
+    nft_id: HexStr
     type: int = field(default_factory=lambda: int(AddressType.NFT), init=False)
 
 
+@json
 @dataclass
 class AccountAddress():
     """An Address of the Account.
     """
     address: str
-    keyIndex: int
+    key_index: int
     internal: bool
     used: bool
 
 
+@json
 @dataclass
 class AddressWithUnspentOutputs():
     """An Address with unspent outputs.
     """
     address: str
-    keyIndex: int
+    key_index: int
     internal: bool
     outputIds: bool
```

Comment:
```suggestion
    output_ids: bool
```
?

Reply:
I've noticed this kind of problems after merging with latest 2.0. Thats why some fields reverted to camelCase